### ISSUE TITLE
pg transition: people/add POST

### ISSUE THREAD

**Issue by JMurphyWeb** [emfoundation/ce100-app#456]
Endpoint: people/add POST

Admin adds a new user.

Fields:
first name*, last name*, job title, phone*, email*, organisation id, user type*

Payload gets sent up as:
```
 { first_name: 'aa',
  last_name: 'aa',
  job_title: '',
  email: 'aa@hh.co',
  phone: '07444444444',
  user_type: 'primary',
  organisation_id: -1 }
```

**Comment by JMurphyWeb** [comment]
I need to update the query to return the organisation name for the purposes of the activation email.

**Comment by SimonLab** [comment]
resolved with #459
